tests/backend/test_execution.py

```diff
@@ -58,34 +58,42 @@ class Mocks:
 
         return mocks
 
-    def test_creates_subprocess_correctly(self, cloud_mocks, mocks):
+    @pytest.mark.parametrize("include_local_env", [True, False])
+    def test_creates_subprocess_correctly(self, cloud_mocks, mocks, include_local_env):
         # Returned a scheduled flow run to start
         cloud_mocks.FlowRunView.from_flow_run_id().state = Scheduled()
         # Return a finished flow run after the first iteration
         cloud_mocks.FlowRunView().get_latest().state = Success()
 
-        execute_flow_run_in_subprocess("flow-run-id")
+        execute_flow_run_in_subprocess(
+            "flow-run-id", include_local_env=include_local_env
+        )
 
         # Should pass the correct flow run id to wait for
         mocks.wait_for_flow_run_start_time.assert_called_once_with("flow-run-id")
 
+        # Merge the starting env and the env generated for a flow run
+        base_env = os.environ.copy() if include_local_env else {}
```

Comment:
I've always had an idiosyncratic preference for `dict()` instead of `{}` because of the visual similarity to set syntax

Reply:
That makes sense, I may do that more in the future.